README.rst

```diff
@@ -69,7 +69,7 @@ Sample ``environment.yml`` is provided below.
      - pip =20
      - pip:
        - odc-stac
-
+       - pystac-client
```

Comment:
`pystac-client` is available on `conda-forge`, should be just below `- pystac` line, and not from `pip`.

Reply:
Done